src/python/lib/ensembl/compara/config/mlss_species_set_adaptor.py

```diff
@@ -0,0 +1,52 @@
+"""
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
```

Comment:
This is something we are going to introduce soon, so you can go ahead:
```suggestion

__all__ = ['get_species_set_by_name']

```